check_openapi_generator_has_been_run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
```

Comment:
Any reason this is in the project's main directory instead of in scripts or .github?

Reply:
No reason at all. It can be moved there

Reply:
Actually, I spoke too early. 
If it is moved to .gihub 
`make` doesn't seem quite happy about it 
```
-t openapi/templates --ignore-file-override ./.openapi-generator-ignore
^Cmake: *** [Makefile:421: openapi/generate/kas-public] Interrupt
```
would you know why?

Reply:
I can think of a possible reason without having looked at it in detail:

Some of the tasks being run are run relative to the current directory so I guess it can't find the files then?

The appropriate approach in that case is to reference paths using `PROJECT_PATH` variable in the `Makefile`

Reply:
I did indeed run the script from a different directory. 
This has been fixed now.